web_service.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/emicklei/go-restful/log"
 )
 
+const DefaultRootPath = "/"
```

Comment:
i don't think introducing an exposed (part of api) const is needed. It will probably never be changed and I would argue it to be a recommended way to globally change routes of an app.